Makefile

```diff
@@ -52,27 +52,28 @@ $(RUNSVINIT): vendor/runsvinit/*.go
 
 $(SCOPE_EXE): $(shell find ./ -path ./vendor -prune -o -type f -name *.go) prog/static.go $(CODECGEN_TARGETS)
 
-report/report.codecgen.go: $(call GET_CODECGEN_DEPS,report/)
+report/latest_map_impls_generated.go: report/latest_map_impls.go tools/generate_latest_map
+report/report.codecgen.go: $(call GET_CODECGEN_DEPS,report/) report/latest_map_impls_generated.go
```

Comment:
Not sure why we need these dep changes? Regardless, adding the `report/latest_map_impls_generated.go` as a dependency looks to be done in the wrong place.